tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -951,7 +978,11 @@ bpf_code_generator::emit_c_code(std::ostream& output_stream)
         if (section.referenced_map_indices.size() > 0) {
             // Emit the array for the maps used.
             std::string map_array_name = program_name.c_identifier() + "_maps";
-            output_stream << format_string("static uint16_t %s[] = {", map_array_name) << std::endl;
+
+            // This line causes an 'Invalid format string' error. Keeping it for further investigation
+            // output_stream << std::vformat("static uint16_t {}[] = {\n", make_format_args(map_array_name));
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#coding-conventions says

> **DON'T** use commented-out code, or code in an #if 0 or equivalent. Make sure all code is actually built.

 Ok to file an issue about this though, and if you like it's ok to add a // TODO comment that references the issue by number.

Reply:
duh.  It was staring me in the face all along :-P  ```std::format``` format specifier delimiter characters (```{```, ```}```) need to be escaped (```{{``` just before the trailing '\n' in our use case) when their literal representation is required.